controllers/machinehealthcheck_controller.go

```diff
@@ -452,3 +475,20 @@ func machineNames(machines []*clusterv1.Machine) []string {
 	}
 	return result
 }
+
+func getMaxUnhealthy(mhc *clusterv1.MachineHealthCheck) (int, error) {
+	if mhc.Spec.MaxUnhealthy == nil {
+		return 0, errors.New("maxUnhealthy not set: maxUnhealthy should always be set")
```

Comment:
```suggestion
	if mhc.Spec.MaxUnhealthy == nil {
		return 0, errors.New("spec.maxUnhealthy must be set")
```

Is this something that's validated or defaulted?